File: Android/divyaagarwal24_DivyaAgarwal_2024it1100_2/GettingStarted/printZ.java


public class printZ {

    public static void main(String[] args) {
        int i,j,k,t=3,x;
        for(i=0;i<5;i++)
        {
            System.out.print("*");
        }
        System.out.println();
        for(j=0;j<3;j++)
        {
            for(k=0;k<t;k++)
            {
            System.out.print(" ");
            }
            t--;
            System.out.println("*");
        }
        for(x=0;x<5;x++)
        {
            System.out.print("*");
        }
        // Write your code here

    }
}

File: Web/Akanksha0401_AkankshaMishra_2024cs1126_2/gettingstarted/printz.cpp
#include<iostream>
using namespace std;
int main(int argc, char** agrc){
    //write your code here
for(int i=0;i<5;i++)
    {
        for(int j=0;j<5;j++)
        {
            if(i==0 || i==4)
                cout<<"*";
            else 
            {
                if(i+j==4)
                    cout<<"*";
                else
                    cout<<" ";
            }
        }
        cout<<endl;
    }
}